.travis.yml

```diff
@@ -36,7 +36,7 @@ install:
   # https://github.com/conda-forge/blas-feedstock/issues/58
   - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION
   - conda activate test-env
-  - conda install -y -q lapack "libblas=*=*netlib" cython>=0.26 future>=0.15 "ipopt=$IPOPT_VERSION" numpy>=1.15 pkg-config>=0.29.2 setuptools>=39.0 six>=1.11
+  - conda install -y -q lapack "libblas=*=*netlib" "ipopt=$IPOPT_VERSION" pkg-config>=0.29.2 setuptools>=39.0
```

Comment:
I'd prefer leaving this as is so we pre-install all dependencies from conda. We could have a separate test that attempts to install using pip installed dependencies.

Reply:
reverted it!